iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/PackageInstallationCacheCheck.java

```diff
@@ -51,12 +51,14 @@ public class PackageInstallationCacheCheck implements IacCheck {
   private static final Predicate<String> isFlag = s -> s.startsWith("-");
   private static final Predicate<String> flagWithNecessaryRmOptions = isFlag.and(containsROrF);
   private static final Predicate<String> flagWithoutContainingNecessaryFlags = isFlag.and(not(flagWithNecessaryRmOptions));
+  private static final Predicate<String> packageName = s -> !s.contains("/") && !s.endsWith(".deb");
```

Comment:
Since we also support `apk`, `.apk` can also be added here as stated in the [docs](https://wiki.alpinelinux.org/wiki/Alpine_Package_Keeper#Add_a_local_Package)